builtin-functions/kphp-light/unsupported/hash.txt

```diff
@@ -30,10 +30,31 @@ function md5_file ($s ::: string, $raw_output ::: bool = false) ::: string | fal
 function sha1 ($s ::: string, $raw_output ::: bool = false) ::: string;
 
 
+define('ZLIB_ENCODING_RAW', -0x0f);
+define('ZLIB_ENCODING_DEFLATE', 0x0f);
+define('ZLIB_ENCODING_GZIP', 0x1f);
+
+define('ZLIB_NO_FLUSH', 0);
+define('ZLIB_PARTIAL_FLUSH', 1);
 define('ZLIB_SYNC_FLUSH', 2);
+define('ZLIB_FULL_FLUSH', 3);
+define('ZLIB_FINISH', 4);
+define('ZLIB_BLOCK', 5);
+define('ZLIB_TREES', 6);
+
+define('ZLIB_FILTERED', 1);
+define('ZLIB_HUFFMAN_ONLY', 2);
+define('ZLIB_RLE', 3);
+define('ZLIB_FIXED', 4);
+define('ZLIB_DEFAULT_STRATEGY', 0);
+
+/** @kphp-generate-stub-class */
+final class DeflateContext {
+    private function __construct() ::: DeflateContext;
+}
 
 /** @kphp-extern-func-info generate-stub */
-function deflate_init(int $encoding, array $options = []) ::: ?DeflateContext;
+function deflate_init(int $encoding, mixed $options = []) ::: ?DeflateContext;
```

Comment:
According to PHP spec, this function accepts array, not mixed.

Reply:
I did it to avoid writing a manual stab for this function, because in case of array it is necessary. I'll return it back

Reply:
So, you want to leave to as is and change it later? If so, please add clarifying comments.